src/lookupStream.js

```diff
@@ -154,6 +149,8 @@ function getCountryCode(result) {
   return undefined;
 }
 
-module.exports = {
-  createLookupStream: createLookupStream
+module.exports = (options) => {
+  maxConcurrentReqs = _.get(options, 'maxConcurrentReqs', 1);
```

Comment:
having these be global variables in this module seems a bit odd to me since you can make many different streams with different vars and the global will always reflect the last ones.